indexservice/indexer.go

```diff
@@ -55,6 +56,20 @@ type (
 		ExecutionHash string
 		BlockHash     string
 	}
+
+	// ActionHistory defines the schema of "action history" table
+	ActionHistory struct {
```

Comment:
if we don't have a field of action type, then how can we filter different type of actions